tests/Type/PHPUnit/AssertFunctionTypeSpecifyingExtensionTest.php

```diff
@@ -16,6 +16,12 @@ public function dataFileAsserts(): iterable
 		}
 
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/assert-function.php');
+
+		if (!function_exists('PHPUnit\\Framework\\assertObjectHasProperty')) {
+			return [];
+		}
+
+		yield from $this->gatherAssertTypes(__DIR__ . '/data/assert-function-9.6.11.php');
```

Comment:
I don't like this structure. I know PHPCS requires it, but I like this more:

```php
		something();
		if (function_exists('PHPUnit\\Framework\\assertObjectHasProperty')) {
			yield from $this->gatherAssertTypes(__DIR__ . '/data/assert-function-9.6.11.php');
		}
		something();
```

You can change the order of other `yield from ...` calls in this method.

Reply:
Yeah, I initially did it differently, but changed to satisfy PHPCS. I'll look at it later, AFK at this moment.